src/binding.cc

```diff
@@ -70,9 +70,6 @@ static_assert(
 static_assert(sizeof(v8::FunctionCallbackInfo<v8::Value>) == sizeof(size_t) * 3,
               "FunctionCallbackInfo size mismatch");
 
-static_assert(sizeof(v8::PropertyCallbackInfo<v8::Value>) == sizeof(size_t) * 1,
-              "PropertyCallbackInfo size mismatch");
```

Comment:
Is `PropertyCallbackInfo` gone in 12.8?

Reply:
its not gone, we're just not concerned with its internals anymore